pkg/generate/app/cmd/newapp_test.go

```diff
@@ -141,7 +141,7 @@ func TestValidate(t *testing.T) {
 		if len(env) != len(c.env) {
 			t.Errorf("%s: Environment variables don't match. Expected: %v, Got: %v", n, c.env, env)
 		}
-		for e, v := range env {
+		for e, v := range env.Map() {
```

Comment:
not sure this is right - shouldn't c.{env,buildEnv,parms} be redefined as non-maps to test ordering?